supabase/functions/updates/index.ts

```diff
@@ -26,6 +40,13 @@ const headersSchema = z.object({
 const bypassRedis = false
 
 async function main(url: URL, headers: BaseHeaders, method: string, body: AppInfos) {
+  let parseResult = null
+  try {
+      parseResult = jsonRequestSchema.parse(body)
```

Comment:
There is something called a “safeParse” you can use that instead of a try catch block

Reply:
See my explanation regarding `safeParse` below.